applications/asset_tracker_v2/tests/ui_module/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(ui_module_test)
+
+# Set CMake path variables for convenience
+set(NRF_SDK_DIR ${ZEPHYR_BASE}/../nrf)
+set(ASSET_TRACKER_V2_DIR ../..)
```

Comment:
Nit: Could use `${NRF_SDK_DIR}/applications/asset_tracker_v2` here to avoid a relative path that causes issues if files are moved

Reply:
On second thought, I believe the reasoning for doing so was in order to run the app tests out of tree. For instance, when
setting up the firmware provided in https://github.com/NordicSemiconductor/asset-tracker-cloud-firmware-aws, the
application folder will be at the same level as the other main repos in NCS.

Reply:
Right, then it makes sense to leave as is